templates/user_profile_page.hbs

```diff
@@ -16,7 +16,28 @@
           {{else}}
             {{user.name}}
           {{/if}}
+          {{#each user.badges}}
+            {{#is category_slug "titles"}}
+              <span class="community-badge community-badge-{{category_slug}}" title="{{description}}">
```

Comment:
As noted [here](https://github.com/zendesk/copenhagen_theme/pull/229#pullrequestreview-471523167), screen reader interaction with these badges are awkward, as it reads the description of the badge before you enter the badge to get the title.

Title attributes are a notorious minefield for a11y, but I get why you went with it here. Maybe an `aria-label` with the name would improve this?